api/views.py

```diff
@@ -626,6 +764,152 @@ def get(self, request, format=None):
             context["bad_request"] = "User Generation with explicit token deprecated. Only token_sha256 allowed."
             return Response(context, status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
```

Comment:
I wonder whether we can offload some lines of code to other files. Maybe we can create `schemas.py`  and import here somehow? It starts to get a bit worrisome to work on such a huge file that happens to be the most critical script on the platform :D

Reply:
Agreed. Good idea. Will split them into `schemas.py`

Reply:
`schemas.py` seemed too generic and ambiguous. Went for `oas_schemas.py` instead. Implying that these schemas are only for the OpenAPI spec definition.